pages/my-dashboard.js

```diff
@@ -89,7 +104,7 @@ export default function MyDashboard(props) {
                       acronym={acronym(card.title)}
                       taskList={taskList}
                       dataCy="task-group-list"
-                      openModal={openModal}
+                      openModal={(link) => openModal('/', betaModal)}
```

Comment:
Same as Ben's first comment, `link` is never used here

Reply:
I've completely rewritten the code.